gammapy/background/tests/test_reflected.py

```diff
@@ -1,17 +1,33 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
 from astropy.tests.helper import pytest
-from ...background import Maps, ReflectedRegionMaker
+from astropy.io import fits
+from astropy.coordinates import SkyCoord, Angle
+from .. import ReflectedRegionMaker, find_reflected_regions_for_event_list
+from ...utils.testing import requires_data
+from ...datasets import gammapy_extra
 
+@requires_data('gammapy-extra')
+def test_ReflectedRegionMaker():
+    exclfile = gammapy_extra.filename('test_datasets/spectrum/dummy_exclusion.fits')
+    exclusion = fits.open(exclfile, hdu = 0)[0]
+    fov = SkyCoord(82.87, 23.24, unit='deg')
+    maker = ReflectedRegionMaker(exclusion, fov)
+    target = SkyCoord(80.2, 23.5, unit='deg')
+    r_on = Angle(0.3, 'deg')
+    regions = maker.compute(target, r_on)
+    assert regions.number_of_regions == 19
 
-@pytest.mark.xfail
-def test_TestReflectedBgMaker():
-    runs = 'TODO'
-    maps = Maps('maps.fits')
-    reflected_bg_maker = ReflectedRegionMaker(runs, maps, psi=2, theta=0.1)
-    total_maps = Maps('total_maps.fits')
-    for run in runs:
-        run_map = total_maps.cutout(run)
-        reflected_bg_maker.make_n_reflected_map(run, run_map)
-        total_maps.add(run_map)
-    total_maps.save('n_reflected.fits')
+@requires_data('gammapy-extra')
+def test_find_reflected_regions_for_event_list():
```

Comment:
It would be very valuable to have tests for reflected regions that check if this works for RADEC and GALACTIC and in the Galactic center region 0 = 360 deg.
This has been a constant source of bugs in the HESS software before Christopher rewrote the region code a few years ago.

If you have the time, adding a test where you make a test mask (using [make_empty_image](https://gammapy.readthedocs.org/en/latest/api/gammapy.image.make_empty_image.html)) centered on (GLON, GLAT) = (0,0) and with something excluded (e.g. a strip along the Galactic plane, or two opposite quadrants), and then computing reflected regions there would be very helpful. But of course it takes time .... you decide if it's useful enough to spend the time now!